system/security/shared/seclib.hpp

```diff
@@ -325,6 +396,7 @@ interface IAuthMap : extends IInterface
     virtual bool shouldAuth(const char * path) = 0;
     virtual ISecResourceList * queryResourceList(const char * path) = 0;
     virtual ISecResourceList * getResourceList(const char * path) = 0;
+    virtual bool addToBackend(interface ISecManager* backend, interface IEspSecureContext* secureContext = nullptr) = 0;
```

Comment:
The fact that the secmgr being passed in is considered the backend is kind of beside the point.  Can we just call this method something like copyResourceList() or something similar?

Reply:
The name can be changed. I was originally trying to limit changes related to this - existing usage of the auth map didn't change. If it's going to change, does it make sense to go "all the way" with the change? The constructor accepts a manager instance, and holds it solely for use by this method. In all existing cases, the map construction and the call of this method occur in the same method. I see no reason for the map to accept (or retain) a reference to the manager except when calling this method.